meta-mender-core/conf/mender-vars.json

```diff
@@ -156,5 +156,6 @@
     "MENDER_CONNECT_USER": "",
     "MENDER_UPDATE_CONTROL_MAP_EXPIRATION_TIME_SECONDS": "",
     "MENDER_UPDATE_CONTROL_MAP_BOOT_EXPIRATION_TIME_SECONDS": "",
-    "MENDER_CI_LOGLEVEL": ""
+    "MENDER_CI_LOGLEVEL": "",
+    "MENDER_RFS_PART_EMPTY_B": ""
```

Comment:
While you're switching name, maybe consider putting this up next to the others with similar names. I know, this isn't fully sorted, but might as well do it for this variable.